_rules/meta-viewport-b4f0c3.md

```diff
@@ -11,7 +11,7 @@ accessibility_requirements:
     passed: further testing needed
     inapplicable: further testing needed
   wcag21:1.4.10: # Reflow (AA)
-    secondary: true
+    secondary: This success criterion is **related** to this rule. This is because a page that cannot be zoomed up to 200% often does not reflow sufficiently either. Most failed examples satisfy this success criterion.
```

Comment:
```suggestion
    secondary: This success criterion is **related** to this rule. This is because a page that cannot be zoomed up to 200% often does not reflow sufficiently either. Most failed examples do not satisfy this success criterion.
```

Reply:
Discussed, leaving this for a different PR.